type_checker.py

```diff
@@ -1,26 +1,26 @@
-from __future__ import annotations
```

Comment:
_Note_:

Passing with `annotations` enabled should a minimal requirement for any proposed solution. We shouldn't include code that will break with Python version update.